mlcomp/tests/pipelines/sagemaker_mnist_prediction.json

```diff
@@ -82,6 +83,14 @@
                         "label": "IAM Role",
                         "description": "An IAM identity that has specific permissions",
                         "editable": true
+                    },
+                    "boto3_logging_level": {
+                        "value": {},
+                        "type": "map<string,string>",
+                        "optional": false,
+                        "label": "Logging level",
```

Comment:
Maybe in the description mention that this is the boto3 logging level?

Reply:
Fixed (Also for the others)